deltachat-ios/Controller/ContactDetailViewController.swift

```diff
@@ -31,7 +30,7 @@ class ContactDetailViewController: UITableViewController {
         let cell = ActionCell()
         cell.accessibilityIdentifier = CellIdentifiers.chat.rawValue
         cell.actionColor = SystemColor.blue.uiColor
-        cell.actionTitle = String.localizedStringWithFormat(String.localized("ask_start_chat_with"), contact.nameNAddr)
+        cell.actionTitle = String.localizedStringWithFormat(String.localized("menu_new_chat"), contact.nameNAddr)
```

Comment:
menu_new_chat has no additional string parameters -> remove `String.localizedStringWithFormat`
Should also a action message be added?

Reply:
i've removed that, thanks.
wrt "action message" - i think the button is just fine, but what exactly is the "action message"?

Reply:
oh, I now see that this title is not within an alert but part of a table view cell :O 
Nevermind.